src/Plivo/Resources/Number/Number.php

```diff
@@ -76,12 +76,6 @@ function __construct(BaseClient $client, array $response, $authId)
             'tollFreeSMSVerification' => $response['toll_free_sms_verification'],
             'renewalDate' => $response['renewal_date'],
         ];
-        if (isset($response['object_id'])) {
-            $this->properties['objectId'] =  $response['object_id'];
-        }
-        if (isset($response['object_type'])) {
-            $this->properties['objectType'] =  $response['object_type'];
-        }
```

Comment:
@shubhankarPlivo Remove the same attributes from the numberGetResponse.json and numberListResponse.json file as well in the php 
cc: @apoorv-plivo